tests/test_tket.py

```diff
@@ -0,0 +1,245 @@
+# Copyright 2020-2022 Cambridge Quantum Computing
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from jax import numpy as jnp, jit, grad, random
+import qujax
+
+from pytket.circuit import Circuit, Qubit
+from pytket.pauli import Pauli, QubitPauliString
+from pytket.utils import QubitPauliOperator
+from pytket.extensions.qujax import tk_to_qujax
+
+
+def _test_circuit(circuit, param):
```

Comment:
Could you add type hints in this function?

Reply:
good call!